packages/synapse-react-client/src/components/PortalFeatureHighlights/PortalFeatureHighlights.tsx

```diff
@@ -0,0 +1,86 @@
+import { Typography, Box, Button, CardMedia, Grid } from '@mui/material'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export type PortalFeatureHighlightsProps = {
+  reverseOrder?: boolean
+  title?: string
+  image?: string
+  buttonText?: string
+  summaryText?: string | React.ReactNode
+  link?: string
+}
+
+const PortalFeatureHighlights = (props: PortalFeatureHighlightsProps) => {
+  const { reverseOrder, image, title, buttonText, summaryText, link } = props
+  return (
+    <Grid
+      container
+      gap={{ xs: '38px', md: '80px' }}
+      padding={{ md: '0 56px 0px 56px' }}
+      justifyContent="center"
+      alignItems={{ md: 'center', lg: 'flex-start' }}
+    >
+      <Grid
+        item
+        order={{ xs: 0, md: reverseOrder ? 1 : 0 }}
+        xs={12}
+        md={5}
+        lg={7.6}
+      >
+        <CardMedia
+          component="img"
+          image={image}
+          style={{
+            borderRadius: '12px',
+            objectFit: 'cover',
+            width: '100%',
+          }}
+        />
+      </Grid>
+      <Grid item xs={12} md={4} lg={3.4}>
+        <Box
```

Comment:
Nit, the MUI `Stack` component would be more appropriate (automatically provides a flex column layout)